modules/checkout/commerce_checkout.module

```diff
@@ -55,6 +61,21 @@ function commerce_checkout_theme_suggestions_commerce_checkout_form(array $varia
 }
 
 /**
+ * Implements hook_preprocess_HOOK().
+ */
+function commerce_checkout_preprocess_commerce_checkout_order_summary(&$variables) {
+  /** @var \Drupal\commerce_order\Entity\OrderInterface $order */
+  $order = $variables['order_entity'];
+  /** @var \Drupal\commerce_order\OrderTotalSummaryInterface $order_total_summary */
+  $order_total_summary = \Drupal::service('commerce_order.order_total_summary');
+  $variables['totals'] = [
```

Comment:
This confused me and I had to do a double check, because I expected totals to be the raw array like in the receipt template. Instead, I got the rendered version. For more power let's pass both totals and rendered_totals, we lose nothing by doing so.